tests/robot-tests/tests/libs/admin/manage-content-common.robot

```diff
@@ -468,12 +468,14 @@ user adds image to accordion section text block with retry
     ...    ${FILES_DIR}${filename}
 
     user scrolls up    300
-    wait until keyword succeeds     ${timeout}    %{WAIT_SMALL} sec    user clicks button    Change image text alternative
+    wait until keyword succeeds    ${timeout}    %{WAIT_SMALL} sec    user clicks button
+    ...    Change image text alternative
     user enters text into element    label:Text alternative    ${alt_text}
     user clicks element    css:button.ck-button-save
     sleep    5
     user scrolls up    100
-    wait until keyword succeeds     ${timeout}    %{WAIT_SMALL} sec    user clicks element    xpath://div[@title="Insert paragraph after block"]
+    wait until keyword succeeds    ${timeout}    %{WAIT_SMALL} sec    user clicks element
+    ...    xpath://div[@title="Insert paragraph after block"]
```

Comment:
Yeah, sounds good but because these are all just aspects of reformatting I didn't want to go about improving stuff that I wasn't touching directly.  Happy to make this change though!  Done.